### DISCUSSION TITLE
Git Resources

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#778] [Technical Resources]
If you have a resource to add, drop it below as a link. If you can add a category or context, we're here for it.

**Example**

## Git
**Git Workflow Diagram**
**Source**: https://twitter.com/NikkiSiapno/status/1593882400983072769?s=20 via @davidalpert 

![Git Workflow
 diagram from NikkiSiapno](https://user-images.githubusercontent.com/34313413/208196736-898e1625-9f7d-4858-a86e-10525a5f755a.png)

**Comment by davidalpert** [comment]
check out @abbeyperini's series on how to deal with `#gitpanic`
- https://dev.to/abbeyperini/series/20421

**Comment by BekahHW** [comment]
From VC today:

- https://ohshitgit.com 
- https://sethrobertson.github.io/GitFixUm/fixup.html
- Check out LinkedIn Learning Git!
- https://graphite.dev/guides/clean-local-branches
- @abbeyperini has some great resources https://dev.to/abbeyperini/series/20421 - [#gitPanic - Merging and Rebasing by Abbey Perini](https://www.youtube.com/watch?v=aFeVS_WUi64)
- https://www.epicweb.dev/tutorials/git-fundamentals
- https://www.git-tower.com/blog/git-cheat-sheet
- https://github.com/so-fancy/diff-so-fancy
- https://www.warp.dev/
